system/dummy_infrastructure/launch/dummy_infrastructure.launch.xml

```diff
@@ -6,7 +7,7 @@
   <arg name="output/state_array" default="/system/v2x/virtual_traffic_light_states"/>
 
   <!-- Parameter -->
-  <arg name="config_file" default="$(find-pkg-share dummy_infrastructure)/config/dummy_infrastructure.param.yaml"/>
+  <arg name="dummy_infrastructure_node_param_file" default="$(find-pkg-share dummy_infrastructure)/config/dummy_infrastructure.param.yaml"/>
```

Comment:
Could you keep the argument name in this PR?
It could be referred by other launch files / commands, and we should keep this out of scope from "rework parameter" activity.

Reply:
Modified.